numpy/core/src/_simd/_simd_inc_data.h.src

```diff
@@ -0,0 +1,91 @@
+#if !NPY_SIMD
+    #error "Not a standalone header, only works through 'simd.dispatch.c.src'"
+#endif
```

Comment:
This is quite a weird pattern, I'd strongly prefer to see something like `#include _simd_inc.h` here, if that works.

Reply:
I guard it the sub-headers via the main header guard definition `_SIMD_SIMD_INC_H_`.